ToolBox/ForexVolumeDownloader/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using QuantConnect.Configuration;
+using QuantConnect.Logging;
+using System;
+using System.Globalization;
+
+namespace QuantConnect.ToolBox.FxVolumeDownloader
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            if (args.Length != 4)
+            {
+                Console.WriteLine("Usage: ForexVolumeDownloader SYMBOLS RESOLUTION FROMDATE TODATE");
+                Console.WriteLine("SYMBOLS = eg EURUSD,USDJPY\n" +
+                                  "\tAvailable pairs:\n" +
+                                  "\tEURUSD, USDJPY, GBPUSD, USDCHF, EURCHF, AUDUSD, USDCAD,\n" +
+                                  "\tNZDUSD, EURGBP, EURJPY, GBPJPY, EURAUD, EURCAD, AUDJPY");
+                Console.WriteLine("RESOLUTION = Minute/Hour/Daily/All");
+                Console.WriteLine("FROMDATE = yyyymmdd");
+                Console.WriteLine("TODATE = yyyymmdd");
+
+#if DEBUG
```

Comment:
Is there another way to achieve this without using the #if directive? I would try to avoid it unless absolutely necessary